src/components/sidebar/helpers/generate-resources-nav-items.ts

```diff
@@ -1,29 +1,23 @@
 import { ProductSlug } from 'types/products'
 
-const DEFAULT_COMMUNITY_FORUM_LINK = 'https://discuss.hashicorp.com/'
+const DEFAULT_COMMUNITIY_FORUM_LINK = 'https://discuss.hashicorp.com/'
```

Comment:
```suggestion
const DEFAULT_COMMUNITY_FORUM_LINK = 'https://discuss.hashicorp.com/'
```
re: @tunzor's point on the typo